### PR TITLE
Add jsch library 0.1.52.wso2v1

### PR DESCRIPTION
**Pull request by vinok88**

### CODE REVIEW

File: jsch/0.1.52.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    </dependencies>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+
+        <snapshotRepository>
```
Comment (thusithathilina) [inline jsch/0.1.52.wso2v1/pom.xml]: remove snapshot distro

File: jsch/0.1.52.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+ ~ KIND, either express or implied.  See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.jcraft</groupId>
```
Comment (thusithathilina) [inline jsch/0.1.52.wso2v1/pom.xml]: group id should be org.wso2.orbot<orginla group id>

File: jsch/0.1.52.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    <name>JSch</name>
+    <description>
+        a pure Java implementation of SSH2
+    </description>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.jcraft</groupId>
+            <artifactId>jsch</artifactId>
+            <version>${version.jsch}</version>
```
Comment (thusithathilina) [inline jsch/0.1.52.wso2v1/pom.xml]: dependencies should be optional

File: jsch/0.1.52.wso2v1/pom.xml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+                        <Export-Package>
+                            com.jcraft.jsch.*;version="${version.jsch}"
+                        </Export-Package>
+                        <Import-Package>
+                            !com.jcraft.jsch.*,
+                            com.jcraft.jzlib;resolution:=optional,
+                            javax.crypto;resolution:=optional,
+                            javax.crypto.interfaces;resolution:=optional,
+                            javax.crypto.spec;resolution:=optional,
+                            org.ietf.jgss;resolution:=optional
```
Comment (Niranjan-K) [inline jsch/0.1.52.wso2v1/pom.xml]: Any particular reason to have these imports as optional?
In import package, the import version range needs to be defined.

File: jsch/0.1.52.wso2v1/pom.xml [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+
+        <snapshotRepository>
+            <id>wso2.snapshots</id>
```
Comment (Niranjan-K) [inline jsch/0.1.52.wso2v1/pom.xml]: Snapshot repository need not be mentioned here.

File: jsch/0.1.52.wso2v1/pom.xml (hunk 2 above)
Comment (Niranjan-K) [inline jsch/0.1.52.wso2v1/pom.xml]: the groupid should be org.wso2.orbit.<original-bundle-groupId>

**Pull request closed without merging**